package.json

```diff
@@ -65,24 +63,31 @@
     "uvu": "^0.5.1",
     "ws": "^7.5.3"
   },
-  "dependencies": {
-    "@nestjs/graphql": "~8.0.2"
-  },
   "peerDependencies": {
     "@apollo/federation": "^0.26.0",
-    "@apollo/gateway": "^0.33.0",
-    "@nestjs/common": "^8.0.2",
-    "@nestjs/core": "^8.0.2",
-    "@nestjs/platform-fastify": "^8.0.2",
-    "altair-fastify-plugin": "4.0.6",
-    "apollo-server-core": "^3.0.0",
+    "@nestjs/graphql": "^8.0.2",
```

Comment:
ok moving `@nestjs/graphql` as peerDep, but I'm not sure of using `^` instead of `~` to match version.
This library uses some internal tools of `@nestjs/graphql` and I had some issues in the past where an internal change was made in a minor and broke it

Reply:
Done + updated dependencies.  Nest.js v8 had an issue with `onModuleInit` ordering and has been fixed.